policyengine_us/parameters/gov/household/household_benefits.yaml

```diff
@@ -0,0 +1,23 @@
+description: The government counts these sources as household benefits.
+values:
+  0000-01-01:
```

Comment:
can we add accurate start date for the programs - earliest we have parameters added in the model

Reply:
added, the latest earliest start dates for relevant parameters for both `household_benefits` and `household_state_benefits` is 2023-01-01